src/openvslam/tracking_module.cc

```diff
@@ -53,6 +53,17 @@ double get_depthmap_factor(const camera::base* camera, const YAML::Node& yaml_no
     }
     return depthmap_factor;
 }
+
+double get_pose_reloc_distance_threshold(const YAML::Node& yaml_node) {
+    spdlog::debug("load maximum distance threshold where close keyframes could be found");
+    return yaml_node["reloc_distance_threshold"].as<double>(0.2);
```

Comment:
Please add the two parameters to https://github.com/OpenVSLAM-Community/openvslam/blob/main/docs/parameters.rst.